src/plugins/cosmos/components/modals/GetStarted/views/LearnMore.tsx

```diff
@@ -0,0 +1,151 @@
+import { useHistory } from 'react-router-dom'
+import rewards from 'assets/rewards.svg'
+import risk from 'assets/risk.svg'
+import withdraw from 'assets/withdraw.svg'
+import { CarouselDots } from 'components/CarouselDots/CarouselDots'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+
+const STEPS_LENGTH = 3
+const STEP_TO_ELEMENTS_MAPPING = {
+  1: {
+    bodies: [
+      'defi.modals.learnMore.bodies.rateFluctuationInfo',
+      'defi.modals.learnMore.bodies.amountStakingInfo',
+      'defi.modals.learnMore.bodies.withdrawInfo'
+    ],
+    header: 'defi.modals.learnMore.headers.aboutStakingRewards',
+    headerImageSrc: rewards
+  },
+  2: {
+    bodies: ['defi.modals.learnMore.bodies.unbondingInfo'],
+    header: 'defi.modals.learnMore.headers.unstaking',
+    headerImageSrc: withdraw
+  },
+  3: {
+    bodies: [
+      'defi.modals.learnMore.bodies.slashingInfo',
+      'defi.modals.learnMore.bodies.partnerInfo'
+    ],
+    header: 'defi.modals.learnMore.headers.risks',
+    headerImageSrc: risk
+  }
+}
+
+type LearnMoreProps = {
+  assetId: string
+}
+
+export const LearnMore = ({ assetId }: LearnMoreProps) => {
+  const history = useHistory()
+  const { cosmosGetStarted } = useModal()
+  // TODO: wire me up, parentheses are nice but let's get asset name from selectAssetNameById instead of this
+  const asset = (_ => ({
+    name: 'Osmo'
+  }))(assetId)
+
+  const { nextStep, prevStep, activeStep } = useSteps({
+    initialStep: 1
+  })
+
+  // activeStep from useSteps() can have an infinite of step, but at runtime, it is effectively restricted to max. 3 in our case
+  // This makes tsc happy regarding exhaustive property access of STEP_TO_ELEMENTS_MAPPING
+  let currentStep = activeStep as 1 | 2 | 3
+
+  const isLastStep = currentStep === STEPS_LENGTH
+  const isFirstStep = currentStep === 1
+
+  const handleNextOrCloseClick = () => {
+    if (isLastStep) return cosmosGetStarted.close()
+
+    nextStep()
+  }
+
+  const handlePrevClick = () => {
+    if (isFirstStep) {
+      return history.goBack()
+    }
+
+    prevStep()
+  }
+
+  return (
+    <>
+      <IconButton
+        variant='ghost'
+        color='white'
+        icon={<ArrowBackIcon />}
+        aria-label={'common.back'}
+        position='absolute'
+        top={2}
+        left={3}
+        fontSize='xl'
+        size='sm'
+        onClick={handlePrevClick}
+      />
+      <Box pt='36px' pb='20px' px='24px'>
+        <Flex
+          direction='column'
+          maxWidth='395px'
+          height='520px'
+          alignItems='center'
+          justifyContent='space-between'
+        >
+          <SlideTransition key={currentStep}>
+            <Flex direction='column' alignItems='center'>
+              <DefiModalHeader
+                headerImageSrc={STEP_TO_ELEMENTS_MAPPING[currentStep].headerImageSrc}
```

Comment:
We can now update all of these, e.g. `headerImageSrc={currentElement.headerImageSrc}` etc

Reply:
Done